src/config/constants.ts

```diff
@@ -29,7 +29,11 @@ export const getProductBrandingCode = () => 'redhat';
 export const POLLING_INTERVAL = 10 * 1000;
 export const EVENTS_POLLING_INTERVAL = 10 * 1000;
 
-export const HOST_ROLES = ['worker', 'master'];
+export const HOST_ROLES: { [key in HostRoleUpdateParams]: string } = {
+  'auto-assign': 'Automatic',
```

Comment:
auto-assign is shown as Automatic.
I've also changed other roles to be upper case